src/Components/Input/Input.js

```diff
@@ -1,10 +1,10 @@
-import React, { Component } from "react";
-import "./Input.css";
-import Validator from "..\\Utils\\Validator";
+import React, { Component } from 'react';
+import './Input.css';
+import Validator from '..\\Utils\\Validator';
```

Comment:
replace `import Validator from '..\\Utils\\Validator';`
to `import Validator from '../Utils/Validator';`

Reply:
fixed